staticServer/router.go

```diff
@@ -2,49 +2,110 @@ package staticServer
 
 import (
 	"io/ioutil"
-	"log"
 	"net/http"
 	"os"
 
 	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
 )
 
-// NewRouter сreates and returns gorilla router
-func NewRouter() *mux.Router {
-	r := mux.NewRouter()
-	r.Handle("/", http.FileServer(http.Dir("../../staticServer/static")))
-	r.PathPrefix("/static/").Handler(http.StripPrefix("/static/", http.FileServer(http.Dir("../../staticServer/static"))))
-	r.Path("/content").HandlerFunc(fetch("fridgeContent")).Methods("GET")
-	r.Path("/recipes").HandlerFunc(fetch("allRecipes")).Methods("GET")
-	return r
+// Path to yaml file, used for server configuration
+const (
+	CfgPath           = "../../clientConfig.yaml"
+	DefaultPort       = ":5080"
+	DefaultStaticPath = "../../staticServer/static"
+	DefaultPrefix     = "/static/"
+)
+
+// Saves configs, extracted from .yaml file
+var serverCfg *config
+
+
+type config struct {
+	Port         string `yaml:"port"`
+	ServerAddres string `yaml:"serverAddress"`
+	Host         string `yaml:"localhost"`
+	StaticFilesPath string `yaml:"staticFilesPath"`
+	Prefix          string `yaml:"pathPrefix"`
+}
+
+// Reads configuration file and stores values to struct variable
+func getSettings(cfgPath string) *config {
+	cfg, err := ioutil.ReadFile(cfgPath)
+	if err != nil {
+		log.WithField("error", err).Error("Failed to read config file")
```

Comment:
If error occurs, you wouldn't be able to continue. Please try to stop processing on as fast as possible.
You have to return error from this function to process it in caller

Reply:
Please dont use `err` as field. Fields are created to store common metadata.
In this particular case it would be better to use `errors.Wrap`

Reply:
I changed to Errorf (logrus)... Didn't want to pull entire package to use one function :sweat_smile: